src/grocker/__main__.py

```diff
@@ -191,11 +191,12 @@ def main():
         )
 
     if GrockerActions.push_img in args.actions:
-        if builders.is_prefixed_image(image_name):
+        if not builders.is_prefixed_image(image_name):
             logger.warning('Not pushing any image since the registry is unclear in %s', image_name)
         else:
             logger.info('Pushing image...')
-            results['hash'] = builders.docker_push_image(docker_client, image_name)
+            image = builders.docker_push_image(docker_client, image_name)
+            collect['hash'] = [x.split('@')[1] for x in image.attrs['RepoDigests']][0]
```

Comment:
This should still be named results in this commit.
The rename only happens with the CLI switch